index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <script src='main.js'></script>
```

Comment:
it would be great if you link this script at the bottom of the body tag.

Reply:
> it would be great if you link this script at the bottom of the body tag.

- [ ] Good job at replying 😃, but I see that you told the student to put the script at the bottom of the body without explaining why to put the script below, so kindly 😊try to tell the student the reason of the change because this may be obvious to you and not clear to the student 💯